docs/pages/markdowns/JavaScript_API.md

```diff
@@ -52,6 +52,37 @@ window.mmgisAPI.updateVectorLayer('Waypoints', {
 
 ```
 
+### trimVectorLayer(layerName, keepAfterTime, timePropPath)
+
+This function removes features on a specified layer before a specified time
+
+#### Function parameters
+
+- `layerName` - name of layer to update
+- `keepAfterTime` - absolute time in the format of YYYY-MM-DDThh:mm:ssZ; will keep all features after this time
+- `timePropPath` - name of time property to compare with the time specified by keepAfterTime
+
+The following is an example of how to call the `trimVectorLayer` function:
+
+```javascript
+window.mmgisAPI.trimVectorLayer('Waypoints',  "2021-12-01T15:10:00.000Z", 'time')

```

Comment:
The addition of milliseconds is slightly different than the format string above, but I think JavaScript handles it all the same anyway.